packages/hamclock-big/hamclock-big.pacscript

```diff
@@ -0,0 +1,45 @@
+pkgname="hamclock-big"
+pkgver="4.00"
+source=("https://github.com/kj7rrv/hamclock/archive/refs/tags/v${pkgver}.tar.gz")
+url=('https://clearskyinstitute.com/ham/HamClock')
+makedepends=("gcc" "make" "pkg-config" "libxau6" "libx11-6" "libx11-dev" "libxcb1")
+breaks=("hamclock" "hamclock-bigger" "hamclock-huge")
+replaces=("hamclock-big")
+pkgdesc="Clock and world map with extra features for amateur radio (1600x960 version)"
+sha256sums=("1def12cf34d3486edb1c4fb0ec7f1a27cc1bda01ca811b6c0fefc68c4c98ab20")
+maintainer=("Roy Williams <fang64@gmail.com>")
+repology=("project: ${pkgname}")
+
+prepare() {
+  cd "${_archive}"
+  cd "ESPHamClock"
```

Comment:
Replace with `cd "hamclock-${version}/ESPHamClock"`

`${_archive}` exists for the 5.0.0 migration, and will be removed in the future, so it's better to replace it now

Reply:
Do I need to format the release files from my github repo? I'd need to change the AUR packages too then.